<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultWindow</class>
 <widget class="QDialog" name="ResultWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>461</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="resList"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="statisticInformation">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>471</width>
     <height>451</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="Ukrainian" country="Ukraine"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>461</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="keyStatus">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="locale">
       <locale language="Ukrainian" country="Ukraine"/>
      </property>
      <property name="text">
       <string>Ключ генерації не буде збережено</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveKey">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="locale">
       <locale language="Ukrainian" country="Ukraine"/>
      </property>
      <property name="text">
       <string>Зберегти ключ генерації</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="nModelsLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>460</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Зібрати статистику за n симуляцій та зберегти у файл</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="saveCsv">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>516</y>
     <width>231</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>формату csv</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveTxt">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>516</y>
     <width>231</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>формату txt</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
